trimmer_decor.rb

```diff
@@ -0,0 +1,6 @@
+require_relative 'base_decor'
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    @nameable.correct_name[0..10]
```

Comment:
- [ ] This method returns 11 characters instead of 10. Kindly fix that. What we should have is "Maximilian", not "Maximilianu". You can fix this by adding one more dot (.) here:

  ```
  @nameable.correct_name[0...10]
  ```

  ![image](https://user-images.githubusercontent.com/36542986/200124741-db1ea07e-a155-44b2-9ab1-d3775e2a3b32.png)

Reply:
Done